packages/react-components/src/checkbox/src/Checkbox.css

```diff
@@ -46,38 +51,34 @@
     left: 50%;
     width: var(--scale-echo);
     height: var(--scale-echo);
+    mask: url("~@orbit-ui/icons/dist/icon-check-24.svg") center center no-repeat;
```

Comment:
I guess this is a new strategy that I don't understand yet but it seems weird to render the check icon when the checkbox is not yet checked.

Reply:
that way the checkmark is already in place and won't have a weird transition